applications/ecallmgr/src/ecallmgr_call_events.erl

```diff
@@ -951,11 +973,21 @@ get_fax_ecm_used(Props) ->
         Else -> Else =/= <<"off">>
     end.
 
+-spec get_serialized_history(wh_proplist()) -> binaries().
+get_serialized_history(Props) ->
+    case kzd_freeswitch:transfer_history(Props) of
+        'undefined' -> [];
+        History when is_binary(History) ->
+            ecallmgr_util:unserialize_fs_array(History);
+        History when is_list(History) ->
```

Comment:
How can history be a list? `kzd_freesiwtch:transfer_history/1` has `api_binary()` as its spec. It needs updating?